dotnet/src/Functions/Functions.OpenApi/RestApiOperationRunner.cs

```diff
@@ -348,15 +357,15 @@ private JsonObject BuildJsonObject(IList<RestApiOperationPayloadProperty> proper
     /// </summary>
     /// <param name="payloadMetadata">The payload meta-data.</param>
     /// <param name="arguments">The payload arguments.</param>
-    /// <returns>The HttpContent representing the payload.</returns>
-    private HttpContent BuildPlainTextPayload(RestApiOperationPayload? payloadMetadata, IDictionary<string, object?> arguments)
+    /// <returns>The text payload and corresponding HttpContent.</returns>
+    private (object?, HttpContent) BuildPlainTextPayload(RestApiOperationPayload? payloadMetadata, IDictionary<string, object?> arguments)
```

Comment:
nit.
Add named tuples to make it easier.

So instead of seeing Item1, Item2, the caller actually sees the provided names `Payload`, `Content` in the IDE.

```suggestion
    private (object? Payload, HttpContent Content) BuildPlainTextPayload(RestApiOperationPayload? payloadMetadata, IDictionary<string, object?> arguments)
```